ethcore/src/client/client.rs

```diff
@@ -2728,7 +2730,9 @@ impl ImportExportBlocks for Client {
 				}
 			}
 		};
-		self.flush_queue();
+		self.importer.block_queue.flush();
+		self.import_verified_blocks();
```

Comment:
what is rationale for this change? the logic has changed here

Reply:
> the logic has changed here

Not really? `flush_queue()` is not used outside tests except here. We know that we have just added blocks to the queue so no need to check it again.

Reply:
As discussed previosly `flush_queue` was doing `import_verified_blocks()` in a loop and we removed the logic here, so I think the method removal should be reverted.

Reply:
or maybe I'm misunderstanding why it was done in a loop previously

Reply:
ok I see, I thought all blocks were imported in `import_verified_blocks()` but it actually has an internal configuration, `max_blocks_to_import` so I think that this should be reverted too. Thus, so we are sure that the `block queue` actually is empty.

Reply:
Yeah, I messed up here. 
I'm still annoyed by having a `pub` that is used only here and from a bunch of tests. I mistakenly thought it was an "important" call for the main importing code path but it's not. I'd probably prefer keeping it test-only and copy the logic here if I was to decide but I know ya'all would hate that so… reverting.